src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './couchdb/as-couch-doc'
 export * from './couchdb/bulk-get'
+export * from './couchdb/couch-error-cleaners'
```

Comment:
Where is this file?

Reply:
Looks like I fat-fingered this. The PR is updated now.